lib/parse-model.js

```diff
@@ -11,20 +11,26 @@ function configureFaker(config) {
 function parseModel(model) {
   const modelKeys = Object.keys(model)
   const parsedModel = modelKeys.reduce((accumulator, currentValue) => {
-    const { type, value } = model[currentValue]
-
-    const _value = modelAttributeTypes[type](value)
+    const { type, value, options } = model[currentValue]
+    const _value = modelAttributeTypes[type](value, options)
 
     const _accumulator = Object.assign(accumulator, { [currentValue]: _value })
     return _accumulator
   }, {})
   return parsedModel
 }
 
+function parseArray(model, options) {
+  return [...Array(options.size).keys()].map(() => {
+    return parseModel(model.value)
+  });
+}
+
 const modelAttributeTypes = {
   randomNumberBetween: numbers.randomBetween,
   faker: (args) => Object.byString(faker, args)(),
-  Object: (args) => parseModel(args)
+  Object: (args) => parseModel(args),
```

Comment:
Now that you're here, you could simplify the function to use the param implicitly :P